### ISSUE TITLE
Add tags for icons to make them readable for TalkBack

### ISSUE THREAD

**Issue by Andepthy** [PranshulGG/WeatherMaster#650]
I tested WeatherMaster with TalkBack, and it seems that TalkBack can't read those icons without a text, such as Settings, Edit, Search... It just says "No tag". May be it should be added a tag or something?

These icons in red boxes for example:
<img src="https://github.com/user-attachments/assets/fa3a7da0-0b64-4518-9a43-bbaeac9a1538" width="200" />

------
TalkBack demostration video (in Chinese, and **the volume is LOUD🔊**):

https://github.com/user-attachments/assets/f15431f3-a014-4354-abf0-c6b5da8a6884

(TalkBack keep saying "No tag"(未加标签) for these icons)

**Comment by Andypsl8** [comment]
> @Andypsl8 Does it work with the back arrow icon? 

Yes
